src/math/casadi.hpp

```diff
@@ -6,6 +6,48 @@
 #define __pinocchio_math_casadi_hpp__
 
 #include <casadi/casadi.hpp>
+// #include <Eigen/Core>
+#include <Eigen/Dense>
+
+// Copy casadi matrix to Eigen matrix
+template <typename MT>
+inline void copy(casadi::SX const& src, Eigen::MatrixBase<MT>& dst)
```

Comment:
@mkatliar These functions should appear in an external namespace

Reply:
Like pinocchio::casadi

Reply:
Agree